packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -7,7 +7,7 @@ const jju = require('jju');
 const testFiles = [
   '**/*{.,-}test.{ts,tsx}',
   '**/*.stories.tsx',
-  '**/{test,tests,stories}/**',
+  '**/{common,test,tests,stories}/**',
```

Comment:
Had to add this to solve this:

![image](https://user-images.githubusercontent.com/14183168/143589653-f480eb14-bbe1-4db5-942a-77c213b5cbc4.png)

If it's not acceptable, I can add override in local `eslintrc`. @Hotell WDYT?

Reply:
I think its ok. how did you get this error ? on CI when running `lint` task for `make-styles`?

Reply:
On CI with `lint` command, was able to repro it locally, too